src/core/core.animation.js

```diff
@@ -93,4 +96,19 @@ export default class Animation {
 
 		me._target[prop] = me._fn(from, to, factor);
 	}
+
+	wait() {
```

Comment:
This lets stuff wait for the animations to finish?

Reply:
> Should other chart types use shared animations as well? The radar chart is very similar to the line chart but doesn't use them

The shared options / animations is a optimization for large datasets, I'm not sure if radar charts have ever that many points (with same styling). But its easy to add if there is a use case.

Reply:
Yes, returns Promise for each animation that then gets resolved / rejected.